app/models/data_source_action.rb

```diff
@@ -0,0 +1,5 @@
+class DataSourceAction < ActiveRecord::Base
+  belongs_to :series
+  has_one :users
```

Comment:
`belongs_to :users`? Or am I missing a subtlety of how we want the ORM stuff to work?

Reply:
oops. thanks